prototype/water-extent-map-vertex.ipynb

```diff
@@ -0,0 +1,124 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import copy\n",
+    "\n",
+    "import ipywidgets as widgets\n",
+    "from IPython.display import display\n",
+    "\n",
+    "import hyp3_sdk"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hyp3_watermap = hyp3_sdk.HyP3('https://hyp3-watermap.asf.alaska.edu', prompt=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "granules = widgets.Textarea(\n",
+    "    placeholder='copy-paste Sentinel-1 granule names or File IDs here (One granule per line)',\n",
```

Comment:
I wasn't familiar with `widgets.Textarea` so my first thought was I'd replace the text *in the cell* with my granules, rather than running the cell as-is then pasting my granules in the text area that appears. Might be no action necessary, or might be worth clarifying in any prose instructions for this cell.

Reply:
I've moved stuff around a bit and I think I've addressed this.